qualtran/resource_counting/_bloq_counts.py

```diff
@@ -115,6 +116,16 @@ def __str__(self):
         return f'{self.gateset_name} counts'
 
 
+FloatRepr_T = Union[str, sympy.Expr]
```

Comment:
we usually don't put an underscore before the `T`

Reply:
fixed